package.json

```diff
@@ -50,8 +50,11 @@
     "@babel/core": "^7.0.0",
     "@expo/samples": "~3.0.3",
     "@expo/vector-icons": "^10.0.3",
-    "@react-native-community/masked-view": "^0.1.1",
+    "@react-native-community/masked-view": "^0.1.7",
     "@react-navigation/core": "^3.0.0-alpha.13",
+    "@react-navigation/drawer": "^5.4.1",
+    "@react-navigation/native": "^5.1.5",
+    "@react-navigation/stack": "^5.2.10",
     "@react-navigation/web": "^1.0.0-alpha.9",
```

Comment:
Nit: I think we should delete out the packages deprecated due to the upgrade such as `react-navigation/core`, `react-navigation/web`, etc https://reactnavigation.org/docs/upgrading-from-4.x/